packages/storage/amplify_storage_s3_dart/test/storage_s3_service/task/part_size_util_test.dart

```diff
@@ -0,0 +1,34 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import 'package:amplify_storage_s3_dart/src/storage_s3_service/service/task/part_size_util.dart'
+    as part_size_util;
+import 'package:test/test.dart';
+
+void main() {
+  group('part_size_util', () {
+    test(
+      'ensure part size to at least be ${part_size_util.minPartSize}',
+      () {
+        const testFileSize = 11 * 1024 * 1024; // 11MiB
+        final result = part_size_util.getOptimalPartSize(testFileSize);
+
+        expect(result, part_size_util.minPartSize);
+      },
+    );
+
+    test(
+      'allow part size to be larger than ${part_size_util.minPartSize}',
+      () {
+        const testFileSize = 5 * 1024 * 1024 * part_size_util.maxNumberOfParts +
+            1000 * 1024 * 1024; // 51GiB
+        final expectedResult =
```

Comment:
Pick known quantities if you're gonna unit test simple functions like this. Otherwise, this doesn't really test anything.

Reply:
Makes sense, will update